run_solve_single.sh

```diff
@@ -0,0 +1 @@
+PYTHONPATH=../legate.core.internal/build/lib/python3.8/site-packages/legate.core-22.3-py3.8.egg/ python3 ../legate.core.internal/build/bin/legate --gpus 1 examples/solve.py -m 2048 -n 2048 --nrhs 512
```

Comment:
don't push a script for your local testing

Reply:
sorry, that was unintentional.